deepchecks/utils/distribution/prediction_drift.py

```diff
@@ -0,0 +1,113 @@
+# ----------------------------------------------------------------------------
+# Copyright (C) 2021-2023 Deepchecks (https://www.deepchecks.com)
+#
+# This file is part of Deepchecks.
+# Deepchecks is distributed under the terms of the GNU Affero General
+# Public License (version 3 or later).
+# You should have received a copy of the GNU Affero General Public License
+# along with Deepchecks.  If not, see <http://www.gnu.org/licenses/>.
+# ----------------------------------------------------------------------------
+#
+"""Module contains the Abstract cass for Prediction Drift checks."""
+import numpy as np
+import pandas as pd
+
+from deepchecks import CheckResult
+from deepchecks.utils.distribution.drift import calc_drift_and_plot, get_drift_plot_sidenote
+
+
+class PredictionDriftAbstract:
```

Comment:
Love the abstract!! 
but, is it possible to work without the context? with it we will not be able to use this abstract class for vision and nlp

Reply:
depending on how much time you have, maybe try and make the nlp prediction drift work with this as well